piety-promoted-v3/modernized/05-chapter-05.adoc

```diff
@@ -867,3 +875,5 @@ Thus, our dear and worthy friend, after a long and well-spent life, finished his
 and we doubt not,
 has entered that glorious immortality of rest and peace prepared for the righteous.
 His remains were interred in Friends`' burial-ground at Hitchin the 23rd.
+
+--
```

Comment:
Not sure if the testimony should end here, or the paragraph above.

Reply:
I think you got it right. 👍